DevWs for SHINA SYSTEM containing containing Zigbee Power Meter

shinasys opened the pull request.

shinasys committed:
DevWs for SHINA SYSTEM containing containing Zigbee Power Meter

KKlimczukS commented on devicetypes/smartthings/zigbee-power-meter.src/zigbee-power-meter.groovy:
```diff
@@ -23,6 +23,7 @@ metadata {
         fingerprint profileId: "0104", deviceId:"0053", inClusters: "0000, 0003, 0004, 0B04, 0702", outClusters: "0019", manufacturer: "", model: "E240-KR080Z0-HA", deviceJoinName: "Energy Monitor" //Smart Sub-meter(CT Type)
         fingerprint profileId: "0104", deviceId:"0007", inClusters: "0000,0003,0702", outClusters: "000A", manufacturer: "Develco", model: "ZHEMI101", deviceJoinName: "frient Energy Monitor" // frient External Meter Interface (develco) 02 0104 0007 00 03 0000 0003 0702 01 000A
         fingerprint profileId: "0104", manufacturer: "Develco Products A/S", model: "EMIZB-132", deviceJoinName: "frient Energy Monitor" // frient Norwegian HAN (develco) 02 0104 0053 00 06 0000 0003 0020 0702 0704 0B04 03 0003 000A 0019
+        fingerprint inClusters: "0000,0001,0003,0020,0B04,0702", outClusters: "0003,0004,0019", manufacturer: "ShinaSystem", model: "PMM-300Z1", deviceJoinName: "SiHAS Energy Monitor" // SIHAS Power Meter
```
You may remove `inClusters` and `outClusters` from the fingerprint and add whole Raw Description in a comment after this fingerprint.

shinasys replied:
I want to know about the energy value in "Energy Meter" capability.
I'm not sure if this is the total energy measured or the sum of 10 minutes of energy by the device.
What does this value mean?

KKlimczukS replied:
@shinasys 
"Energy Meter" capability is for handling [Wh] (or [kWh]) reports.
"Power Meter" capability is for handling [W] reports.

shinasys replied:
> @shinasys "Energy Meter" capability is for handling [Wh] (or [kWh]) reports. "Power Meter" capability is for handling [W] reports.
I know the difference between Energy Meter and Power Meter.  
What I want to know is whether the value of the Energy Meter is the total cumulative value of this device.
Or I wonder what it is.

KKlimczukS replied:
Usually it should be total energy measured, but You can check it yourself by switching on the device (with some load) for a certain amount of time and see what it reports.

shinasys committed:
Update zigbee-power-meter.groovy

Remove inClusters and outClusters from the fingerprint and add whole Raw Description in a comment after this fingerprint.

shinasys committed:
Update zigbee-power-meter.groovy

Add "zigbee.configureReporting(zigbee.SIMPLE_METERING_CLUSTER, ATTRIBUTE_READING_INFO_SET, DataType.UINT48, 1, 600, 1)"

KKlimczukS commented on devicetypes/smartthings/zigbee-power-meter.src/zigbee-power-meter.groovy:
```diff
@@ -130,14 +138,19 @@ def configure() {
-private getPowerDivisor() { isFrientSensor() ? 1 : 1000 }
-private getEnergyDivisor() { isFrientSensor() ? 1 : 1000 }
+private getActivePowerDivisor() { isPMM300Z1() ? 1 : 10 }
+private getPowerDivisor() {  (isFrientSensor() || isPMM300Z1()) ? 1 : 1000 }
```
Fix spacing here (one space is enough).

shinasys committed:
Update zigbee-power-meter.groovy

Fix spacing.
